package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-form-master-2000",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Since I removed use of `{{bind-attr}}`, consumers of this app will need to be on at least Ember 1.11. For this reason, I bumped the version so those on < 1.11 can stick with `^0.1.0`.